lib/livebook/runtime.ex

```diff
@@ -1101,4 +1101,10 @@ defprotocol Livebook.Runtime do
   """
   @spec unregister_clients(t(), list(client_id())) :: :ok
   def unregister_clients(runtime, client_ids)
+
+  @doc """
+  Fetches the running Kino Proxy's pid from runtime.
+  """
+  @spec fetch_kino_proxy(t()) :: {:ok, pid()} | {:error, :not_found}
+  def fetch_kino_proxy(runtime)
```

Comment:
@jonatanklosko how do you want to tackle this? Do you prefer a `fetch_kino_proxy` callback, as implemented here, or should we just use `GenServer.where({Kino.Proxy, runtime_node})` from Livebook app?

Reply:
It should definitely be a part of the runtime protocol (and we need to ask `Session` anyway, since it has the runtime).

We want to avoid the name `kino`. Let's make it `fetch_proxy_handler` (corresponding to `Livebook.Proxy.Handler`).